android/src/main/java/com/RNAppleAuthentication/AppleAuthenticationAndroidModule.java

```diff
@@ -150,7 +150,7 @@ public void configure(ReadableMap configObject) {
             .clientId(clientId)
             .redirectUri(redirectUri)
             .responseType(SignInWithAppleConfiguration.ResponseType.ALL)
```

Comment:
```suggestion
            .responseType(responseType)
```

as long as we're in here - appears the exact same oversight happened with responseType
- it gets a default value at the top in var responseType
- the config is checked to see if it exists, and if it exists then the config value is set into responseType
- then all that is completely ignored and `ALL` is used

worth fixing? What do you think?

Reply:
I explained [here](https://github.com/invertase/react-native-apple-authentication/issues/362#issuecomment-2776387504) my reasoning related to `responseType`. Fixing it is not as simple as that...